### PR TITLE
Doc/more documentation

### PR DESCRIPTION
**Pull request by stefanheyder**
Some more documentation for `{euclidean,matrices,grassmannian}.py`.

### CODE REVIEW

**Comment by stefanheyder** [comment]
See also #398.

File: geomstats/geometry/matrices.py [hunk 1]
```diff
@@ -22,15 +22,24 @@ def __init__(self, m, n):
         self.metric = MatricesMetric(m, n)
 
     def belongs(self, point):
-        """Check if point belongs to the Matrix space."""
+        """Check if point belongs to the Matrix space.
```
Comment (ninamiolane) [inline geomstats/geometry/matrices.py]: "Evaluate if a point ..." to be consistent with other docstrings? :/ Or should we put "Check" everywhere? It's also a detail.

File: geomstats/geometry/matrices.py [hunk 2]
```diff
@@ -22,15 +22,24 @@ def __init__(self, m, n):
...
+
+
+        Parameters
+        ----------
+        point : array-like, shape=[n_samples, (m,n)]
```
Comment (ninamiolane) [inline geomstats/geometry/matrices.py]: Descriptions?

File: geomstats/geometry/matrices.py [hunk 3]
```diff
@@ -48,8 +57,7 @@ def equal(mat_a, mat_b, atol=TOLERANCE):
 
     @staticmethod
     def mul(*args):
-        """
-        Return the product of matrices a1, ..., an.
+        """Calculate the product of matrices a1, ..., an.
```
Comment (ninamiolane) [inline geomstats/geometry/matrices.py]: Should we use "Calculate"/ "Compute" to describe the functions? I had deleted them all in my PR, to save space and also bcs I thought it was not essential, but happy to put them back everywhere.

File: geomstats/geometry/matrices.py [hunk 4]
```diff
@@ -128,7 +133,20 @@ def make_symmetric(cls, mat):
...
-        """Generate n samples from a uniform distribution."""
+        """Generate n samples from a uniform distribution.
+
+        Each entry in the matrix is drawn from a uniform distribution with
+        support [0,1]
+
+        Parameters
+        ----------
+        n_samples : int
+            amount of samples to generate
```
Comment (ninamiolane) [inline geomstats/geometry/matrices.py]: Should the first letter be upper-case or lower-case? I had used upper case, but happy to change my PR!

File: geomstats/geometry/matrices.py [hunk 5]
```diff
@@ -143,7 +161,19 @@ def __init__(self, m, n):
...
+        Parameters
+        ----------
+        tangent_vec_a : array-like, shape=[n_samples, m, n]
+        tangent_vec_b : array-like, shape=[n_samples, m, n]
+        base_point : array-like, shape=[n_samples, m, n], optional
+
+
+        Returns
+        -------
+        inner_prod : the Frobenius inner product of a and b
```
Comment (ninamiolane) [inline geomstats/geometry/matrices.py]: array-like, shape=...

File: geomstats/geometry/matrices.py [hunk 6]
```diff
@@ -143,7 +161,19 @@ def __init__(self, m, n):
...
 
     def inner_product(self, tangent_vec_a, tangent_vec_b, base_point=None):
-        """Compute Frobenius inner product of two tan vecs at `base_point`."""
+        """Compute Frobenius inner product of two tan vecs at `base_point`.
+
+        Parameters
+        ----------
+        tangent_vec_a : array-like, shape=[n_samples, m, n]
+        tangent_vec_b : array-like, shape=[n_samples, m, n]
+        base_point : array-like, shape=[n_samples, m, n], optional
```
Comment (ninamiolane) [inline geomstats/geometry/matrices.py]: Descriptions? You can copy paste from PR #425 , I have docstrings for inner products there.

**Comment by ninamiolane** [comment]
Flake8 needs to pass

File: geomstats/geometry/matrices.py (hunk 1 above)
Comment (ninamiolane) [reply geomstats/geometry/matrices.py]: I think `Test if a point...` would be best, is has a boolean feel to it.

**Comment by ninamiolane** [comment]
According to https://numpydoc.readthedocs.io/en/latest/format.html (thanks @cshewmake2 ) and follow-up conversation on slack:

Parameters' description starts with uppercase, ends with a .
optional is on the same line as the parameters' type
spaces around :

**Comment by ninamiolane** [comment]
@stefanheyder any update on this?

**Pull request closed without merging**